lib/gollum-lib/wiki.rb

```diff
@@ -886,7 +886,7 @@ def default_committer_name
     # Returns the String email address.
     def default_committer_email
       @default_committer_email ||= \
-        @repo.config['user.email'] || self.class.default_committer_email
+        @repo.config['user.email'].delete('<>') || self.class.default_committer_email
```

Comment:
This is causing the tests to fail in cases when `@repo.config['user.email]` is nil